app/Http/Controllers/TransactionFLowModelController.php

```
@@ -0,0 +1,77 @@
+                if ($nextFlow) {
+                    $nextFlow->update(['status' => TransactionFlowStatus::IN_PROGRESS->value]);
+                }
+                $transactionFlow->refresh();
+            }
+            return response()->json([
+                'message' => 'Transaction Flow Updated Successfully'
+            ], 200);
+        } catch (\Exception $e) {
+            return response()->json(['error' => 'Update failed: ' . $e->getMessage()], 500);
+        }
+    }
```

PHPMD (2.15.0)

The method update() has a Cyclomatic Complexity of 15. The configured cyclomatic complexity threshold is 10. (Code Size Rules)

(CyclomaticComplexity)